fdbclient/ServerKnobs.cpp

```diff
@@ -169,6 +169,11 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( PRIORITY_ENFORCE_MOVE_OUT_OF_PHYSICAL_SHARD,           960 ); if( randomize && BUGGIFY ) PRIORITY_ENFORCE_MOVE_OUT_OF_PHYSICAL_SHARD = 360; // Set as the lowest priority
 
 	// Data distribution
+	init( AVAILABLE_SPACE_PIVOT_PERCENT,                          0.6);
+	init( READ_LOAD_PIVOT_PERCENT,                                0.8);
```

Comment:
If the knobs are `*PERCENT`, then I think the value should be between 0 - 100 (not 0-1) to avoid any confusion. (The code that uses this can then divide it by 100.) Alternately, if a value between 0-1 is preferred, then it should probably be called `*RATIO`.